src/Database/Behaviors/Purgeable.php

```diff
@@ -1,13 +1,16 @@
 <?php namespace Winter\Storm\Database\Behaviors;
 
+/**
+ * @deprecated 1.2.0. We recommend using the \Winter\Storm\Database\Traits\Purgeable trait instead.
+ */
```

Comment:
Why?

Reply:
I'm not really sure why anymore. I think it was when I was railing against behaviours, but you're right, they have their use. We should probably make this share functionality with the trait if possible though, so we don't have to modify code in two places.

Reply:
I think we tried when we first added it and it wasn't that easy to do.